checker/examples/units-extension/Makefile

```diff
@@ -1,24 +1,39 @@
-FILES=Frequency.java \
-  FrequencyRelations.java \
-  Hz.java \
-  kHz.java
+FILES=qual/Frequency.java \
+  qual/FrequencyRelations.java \
+  qual/Hz.java \
+  qual/kHz.java
 
 JAVAOPTS=-source 7 -target 7
 
 JSR308JAVAC?=../../bin/javac
 
-all: framework-test clean
+# gets the full path to the directory of the make file, which is also the root dir of the qual folder
+# for custom projects, it is best to encode the full root path as a variable
+QUALDIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
```

Comment:
This is the directory that contains the "qual" subdirectory, right?
QUALDIR sounds more like it is the "qual" directory.